Library/Formula/influxdb.rb

```diff
@@ -132,4 +145,13 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  def caveats; <<-EOS.undent
+    Config files from old InfluxDB versions are incompatible with version 0.9.
+    If upgrading from a pre-0.9 version, the new configuration file will be named:
+        #{HOMEBREW_PREFIX}/etc/influxdb.conf.default
```

Comment:
This can just be `#{etc}/influxdb.conf.default`. They'll resolve to the same path.